presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -171,6 +175,19 @@ public QueryManagerConfig setQueryManagerExecutorPoolSize(int queryManagerExecut
     }
 
     @NotNull
+    public DataSize getQueryMaxDataSize()
+    {
+        return maxQueryDataSize;
+    }
+
+    @Config("query.max-data-size")
```

Comment:
data size can be taken as many things. Should we be more specific here, like max-input-data-size or max-raw-input-data-size?